script.js

```diff
@@ -1,4 +1,72 @@
+var generateRandomInteger = function () {
+  var randomDec = Math.random() * 3;
+  var randomNum = Math.ceil(randomDec);
+
+  console.log(randomNum);
+  return randomNum;
+};
+
+var randomScissorPaperStone = function () {
+  var Num = generateRandomInteger();
```

Comment:
Try to avoid using shorthands/abbreviations when naming variables, anticipate the scenario where the person reading your code does not understand the shorthands you're familiar with.